### ISSUE TITLE
Creating a client service using only an API key

### ISSUE THREAD

**Issue by lxalln** [IsmailHassani/Mollie.Net#25]
The documentation states that you can create an instance of `ClientService` using only an API key:

`IClientService clientService = new ClientService("{yourApiKey}");`

However, this no longer works. (2019.1291.1)

I am using Mollie with OAuth, and the current constructor for `ClientService` is really not very friendly when using Access Tokens received from Mollie.

Do you have any suggestions on how best to continue? The pattern of using the DI container isn't appropriate here, as the API key is not stored in options, but is stored per tenant.

At the moment, I am going to have to create a `ClientService` manually and do poor man's injection for the extra services it requires:

```csharp
var optionsWrapper = new OptionsWrapper<AppSettings>(new AppSettings { ApiToken = connection.AccessToken });

var clientService = new ClientService(optionsWrapper, _httpClientFactory, _jsonConverterService, _validatorService);
```

If I were able to set the API Key, that would be acceptable in a worst case scenario.

**Comment by IsmailHassani** [comment]
You're correct. OAuth token is at this moment not implemented in the constructor. I have to create an additional service where this info is stored. and can be used through out the library.
